src/betterproto/__init__.py

```diff
@@ -1191,15 +1179,15 @@ def timestamp_to_json(dt: datetime) -> str:
         if (nanos % 1e9) == 0:
             # If there are 0 fractional digits, the fractional
             # point '.' should be omitted when serializing.
-            return result + "Z"
+            return f"{result}Z"
         if (nanos % 1e6) == 0:
             # Serialize 3 fractional digits.
-            return result + ".%03dZ" % (nanos / 1e6)
+            return f"{result}.%03dZ" % (nanos / 1e6)
```

Comment:
I'm not too sure how to make this completely and f-string if anyone knows how to do it feel free to commit it.

Reply:
If I'm not mistaken `f"{result}.{ int(nanos // 1e6) :03d}Z"` is what you're looking for.